frontend/src/reusable/urls.js

```diff
@@ -0,0 +1,3 @@
+const BASE_URL = 'http://localhost:8080'
```

Comment:
❗❗ This need to be changed cause it makes it impossible to use the FE with the BE.

```suggestion
const BASE_URL = 'https://tender-cori-4354e6.netlify.app'
```